rotkehlchen/tests/api/blockchain/test_base.py

```diff
@@ -1817,3 +1827,68 @@ def test_edit_blockchain_account_blank_label(
             cursor=cursor,
             filter_query=AddressbookFilterQuery.make(),
         )[0]) == 0  # addressbook entry has been deleted
+
+
+def test_remove_solana_address_with_transactions(rotkehlchen_api_server: 'APIServer') -> None:
+    """Test that removing a Solana address properly deletes transactions and history events"""
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    solana_tx_db = DBSolanaTx(rotki.data.db)
+    events_db = DBHistoryEvents(rotki.data.db)
+    txs, address1, _, _ = create_test_solana_transactions()
```

Comment:
```suggestion
    txs_for_addy1, address1, _, _ = create_test_solana_transactions()
```
keep the names consistent with eachother